Description: Replace 3 statements with 1 in Python.Set_Default_Console.
 Py_Single_Input is limited to one statement and says:
 SyntaxError: multiple statements found while compiling a single statement
 This fixes a non-fatal error in gnat-gps.
Author: Nicolas Boulenguez <nicolas@debian.org>

--- a/src/python/gnatcoll-scripts-python.adb
+++ b/src/python/gnatcoll-scripts-python.adb
@@ -3711,9 +3711,8 @@
       else
          Cons := Run_Command
            (Script,
-            "sys.stdout = sys.__stdout__" & ASCII.LF
-            & "sys.stdin  = sys.__stdin__" & ASCII.LF
-            & "sys.stderr = sys.__stderr__",
+            "sys.stdout, sys.stdin, sys.stderr = "
+              & "sys.__stdout__, sys.__stdin__, sys.__stderr",
             Hide_Output => True,
             Need_Output => False,
             Errors      => Errors'Access);
